### ISSUE TITLE
The tooltip of controls present on 'Iris the Bot' dialog is not hover able.

### ISSUE THREAD

**Issue by Kirank915** [microsoft/BotFramework-WebChat#5109]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

If the tooltip of controls present on 'Iris the Bot' dialog is not hover able and when large pointers have been selected via platform settings or assistive technology, the pointer can obscure a significant area of the additional content. This will impact the users who utilize screen reader feedback on mouse interactions.

### What browsers and screen readers do this issue affect?

Windows: Chrome with NVDA

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### What is the public URL for the website?

Request Contributors - POC (sharepoint.com)

### How to reproduce the issue?

Step 1: Open the above URL in New Edge/Chrome browsers with valid credentials.
Step 2: Invoke the 'Ask iris now' button which is present under the questions section. Iris the bot dialog will appear. Navigate to the hamburger menu and hover the mouse. Tooltip will appear.
Step 3: Verify that the Tooltip of controls present on 'Iris the Bot' dialog is hover able or not.

### What do you expect?

Tooltip of controls present on 'Iris the Bot' dialog should be hover able.

### What actually happened?

The tooltip of controls present on 'Iris the Bot' dialog is not hover able.

### Do you have any screenshots or recordings to repro the issue?

<img width="982" alt="MAS 1 4 13_Tooltip of controls present on 'Iris the Bot' dialog is not hover able" src="https://github.com/microsoft/BotFramework-WebChat/assets/117266902/5ee08da4-69be-4e56-8eb8-d17564162423">

### MAS reference

https://aka.ms/MAS1.4.13

### WCAG reference

https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html

**Comment by Kirank915** [comment]
#A11yMAS;#A11ySev3;#MAS1.4.13;#HCL;#Accessibility;#FeatureBotframeworkwebchat-Apr23;

**Comment by Kirank915** [comment]
Bug copied from MCAPS: [ISSUE 1767689](https://dev.azure.com/worldwidelearning/WWL%20Accessibility/_workitems/edit/1767689)

**Comment by Kirank915** [comment]
As per the above comment, this issue is not under bot framework team. Hence closed the bug and updated to partner team (WWL).
